libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.m

```diff
@@ -0,0 +1,74 @@
+//
+//  NBGeocoderMetadataParser.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBGeocoderMetadataParser.h"
+
+@implementation NBGeocoderMetadataParser {
+@private
+  SQLiteDatabaseConnection *_databaseConnection;
+}
+
+- (instancetype)init {
+  self = [super init];
+  return self;
+}
+
+- (void)convertFileToSQLiteDatabase:(NSString *)completeTextFilePath
+                       withFileName:(NSString *)textFileName
+                       withLanguage:(NSString *)languageCode {
+  NSString *fileName = completeTextFilePath;
+  NSData *data = [NSData dataWithContentsOfFile:fileName];
+  NSString *myStr = [[NSString alloc] initWithData:data
+                                          encoding:NSUTF8StringEncoding];
```

Comment:
You can combine these into:
`[[NSString alloc] initWithContentsOfFile:fileName encoding:NSUTF8StringEncoding error:error]`

Reply:
fixed.